defender/downloaded/cToken_Monitor_ABI.json

```diff
@@ -1 +1,1052 @@
\ No newline at end of file
+[
```

Comment:
This is fine, although these ABI files are created programmatically if we pull them down from the Defender account.

I'll make a ticket for updating the ABI download to write them to file with the appropriate formatting.